### ISSUE TITLE
Support for compound keys

### ISSUE THREAD

**Issue by davidski** [cynkra/dm#3]
Are there any plans/thoughts on supporting compound keys? I frequently have a table where the primary key is a combination of fields. It looks like putting a PK constraint on something like that in `dm` requires creating a new single column that is a mutation of the two non-unique components.

Thanks for the package!

**Comment by krlmlr** [comment]
Yes, compound keys would be very nice to have, not sure when this happens though. At the very least we need another refactoring of internal structures (#57).

**Comment by krlmlr** [comment]
Blocked by #54, but we're a step closer.

**Comment by krlmlr** [comment]
Now tracked in https://github.com/krlmlr/dm/projects/2.

**Comment by krlmlr** [comment]
Support will be added in dm 0.2.0, planning to send it to CRAN this week.
